docs/contributors/overview.md

```diff
@@ -1,90 +1,58 @@
-    - Analysis scripts.
-- People with:
-    - Basic programming knowledge,
-    - Basic Git familiarity,
-    - Basic understanding of electronics.
+- Basic programming knowledge
+- Basic Git familiarity
+- Very basic understanding of electronics/circuitry
 
 ---
 
 ## High-Level Architecture
 
 At a very high level, the FIRM ecosystem includes:
 
-- **Hardware** – the FIRM PCB and related circuitry (powered by microcontroller, sensors, power stages, etc.).
-- **Firmware** – code running on the microcontroller: sensor drivers, data logging, communication.
-- **Web App** – a React/TypeScript app that talks to FIRM over serial (via Web Serial + WASM parser).
-- **Analysis tooling** – Python/Rust/other scripts that process data from flights.
-
-Later pages should include diagrams and mermaid charts that show:
+- **Hardware** - the FIRM PCB and related circuitry (a microcontroller, sensors, other electrical components, etc.).
+- **Firmware** - code running on the microcontroller: sensor drivers, data logging, communication.
+- **Connected Device** - a device that connects to FIRM and receives data (e.g. a laptop, Raspberry Pi, etc.).
 
-- How data flows from sensors → firmware → storage → web app → CSV/plots.
-- How different repos and tools relate to each other.
+TODO: maybe make a diagram here?
 
 ---
 
 ## Getting Started as a Contributor
 
-If you’re new to FIRM, a good sequence is:
+If you're new to FIRM, you should probably first **[skim the user docs](../users/getting-started.md)** so you know what the device looks like from an end user's perspective.
 
-1. **Skim the user docs**  
-   – [Using FIRM – Quick Start](../users/getting-started.md)  
-   so you know what the device looks like from an end user’s perspective.
+Next, if you're interested in programming, you should follow **[these steps](environment-setup.md)** to set up your development environment.
 
-2. **Set up your development environment**  
-   – [Environment Setup (VS Code, Python, Git)](environment-setup.md)  
-   so your laptop is ready for FIRM development.
-
-3. **Pick an area of focus**:
-    - Hardware / KiCad → see [Hardware & KiCad](hardware-kicad.md)
-    - Firmware / Web app → see [Firmware & Web App Workflow](dev-workflow.md)
-
-4. Join the relevant channels (Discord, email lists, etc.) and start with small issues or documentation improvements.
+Or if you're interested in the electronics/PCB design, you should follow **[the steps](hardware-kicad.md)** to get KiCad and learn our design process/practices.
 
 ---
 
-## Documentation as a Living Resource
+## Keeping our Docs Up-to-Date
 
-Because FIRM touches so many technologies, **no one person remembers everything**. These docs are meant to:
-
-- Capture **hard-won knowledge** (things we had to debug the hard way).
-- Stay **up to date** with the current hardware revision and firmware.
-- Make onboarding new contributors much easier.
+Because FIRM touches so many technologies, **no one person remembers everything**.
```

Comment:
Now this sentence by itself doesn't have meaning and the next paragraph isn't assigning it any meaning either